File: dataset_reduzido/avatar/Python/Code/codeforces_379_A.py
import re
candeleIntere , b = map ( int , input ( ).split ( ) )
s = 0
restoSciolte = 0
while candeleIntere > 0 or restoSciolte >= b :
    candeleIntere += restoSciolte // b
    restoSciolte %= b
    s += candeleIntere
    restoSciolte += candeleIntere % b
    candeleIntere //= b
print ( s )

File: output/LLaMa/avatar/Python/Java/codeforces_379_A.java
import java.util.Scanner;

public class codeforces_379_A {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        System.out.print("Enter two integers separated by space: ");
        String[] input = scanner.nextLine().split(" ");
        int candlesIntere = Integer.parseInt(input[0]);
        int b = Integer.parseInt(input[1]);
        int s = 0;
        int restoSciolte = 0;
        while (candlesIntere > 0 || restoSciolte >= b) {
            candlesIntere += restoSciolte / b;
            restoSciolte %= b;
            s += candlesIntere;
            restoSciolte += candlesIntere % b;
            if (b == 1) {
                break;
            }
            candlesIntere /= b;
        }
        System.out.println(s);
    }
}